addons/udes_stock/views/stock_picking.xml

```diff
@@ -109,10 +109,10 @@
 
             <!-- Change colour of the rows depending on new values -->
             <xpath expr="//field[@name='move_lines']/tree" position="attributes">
-                <attribute name="decoration-danger">ordered_qty &lt; quantity_done</attribute>
+                <attribute name="decoration-danger">ordered_qty &lt; quantity_done or ((ordered_qty > quantity_done) and (state == 'done'))</attribute>
```

Comment:
I don't see why it is different `ordered_qty &lt; quantity_done` should also cover `((ordered_qty > quantity_done) and (state == 'done'))`. The only difference is `&lt;` vs `>`. Have we tried changing that? or do we need both?